src/swap/defi/uni-v2-based/uniV2Contracts.js

```diff
@@ -19,13 +19,22 @@ export const provider: Provider = new ethers.providers.JsonRpcProvider(
 // Contracts
 //
 
+// Routers
 const SPOOKYSWAP_ROUTER_ADDRESS = '0xF491e7B69E4244ad4002BC14e878a34207E38c29'
 export const spookySwapRouter: Contract = new ethers.Contract(
   SPOOKYSWAP_ROUTER_ADDRESS,
   UNISWAP_V2_ROUTER_ABI,
   provider
 )
 
+const TOMBSWAP_ROUTER_ADDRESS = '0x6D0176C5ea1e44b08D3dd001b0784cE42F47a3A7'
+export const tombSwapRouter: Contract = new ethers.Contract(
+  TOMBSWAP_ROUTER_ADDRESS,
+  UNISWAP_V2_ROUTER_ABI,
+  provider
+)
+
+// Wrapped Tokens
```

Comment:
Couldn't we just have a factory function to make this contract type given an address? Then we could parameterize the plugin with either the address or the contract returned from the factory (similar to how we do it in stake-plugins).